test/json-ld.net.tests/JsonFetcher.cs

```diff
@@ -22,8 +22,8 @@ public JToken GetJson(JToken j, string rootDirectory)
                     return JToken.ReadFrom(jreader);
                 }
             }
-            catch (Exception e)
-            { // TODO: this should not be here, figure out why this is needed or catch specific exception.
+            catch (JsonReaderException)
```

Comment:
Experimentally validated that only JsonReaderExceptions routinely occur here, so scoped down this exception.